client/lib/posts/post-list-cache-store.js

```diff
@@ -34,7 +34,11 @@ function getCacheKey( options ) {
 }
 
 function get( query ) {
-	var key = getCacheKey( query );
+	// @TODO disable post-list-cache-store
+	console.warn( 'Disable post-list-cache-store. Development purpose.' );
+	return null;
```

Comment:
I've disabled the cache just for development. I think it could be removed with this approach.
cc @rralian